submissions/maryOkosun/javascript/script.js

```diff
@@ -0,0 +1,12 @@
+var productList = [
+	{productName:"Tecno Spark",imageUrl:"https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1511707171634-5f897ff02aa9%3Fixlib%3Drb-1.2.1%26ixid%3DeyJhcHBfaWQiOjEyMDd9%26w%3D1000%26q%3D80&imgrefurl=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fmobile-phone&tbnid=LVNFDi1BCzVPKM&vet=12ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygAegUIARCgAQ..i&docid=iPam8aYHdwb-BM&w=1000&h=1000&q=phones%20images%20download&ved=2ahUKEwiG0qSyi4LtAhUIEhoKHa3kDoAQMygAegUIARCgAQ", productPrice:"$250"},
```

Comment:
This line of code is too long, kindly break them done and format it properly. Same goes for others

Reply:
I have effected the changes. kindly review, thanks.